ref/Microsoft.Build.Framework/net/Microsoft.Build.Framework.cs

```diff
@@ -155,7 +155,9 @@ public abstract partial class CustomBuildEventArgs : Microsoft.Build.Framework.L
     public abstract partial class EngineServices
     {
         public const int Version1 = 1;
+        public const int Version2 = 2;
```

Comment:
How are we defining these versions? Is it one/release, or a new version for every change? I kinda expected this to collapse into `Version1` since that's "the 17.0 version".

Reply:
> I kinda expected this to collapse into Version1 since that's "the 17.0 version".

Aha! You're right, it makes perfect sense for the granularity to be per-dot-release. Assuming this goes into 17.0 which I was not sure about since Preview5 has a high bar and this is not really a bug fix. I'll revert back to "Version1" if this kind of change is clear to land in 17.0. Thank you!

Reply:
Reverted.